doc/source/template_docker_nextflow.config

```diff
@@ -17,32 +21,32 @@ process {
     // Specify memory and number of threads for processes submitted to the cluster 
     // It is advised to provide higher computational power to wgdParalogs and wgdOrthologs 
     withName: 'wgdParalogs' {
-        cpus =               // e.g. 8
-        penv = ''            // e.g. 'serial'
-        memory = ''          // e.g. '16GB'
-        clusterOptions = ''  // e.g. '-l h_vmem=2G'
+        cpus =                  // e.g. 8
```

Comment:
Maybe change the comment to something like "e.g. 8 on a cluster or 2 on local computer"? Same for below.

Reply:
Agreed!